### PR TITLE
chore(package): update deps

### PR DESCRIPTION
**Pull request by marcoscaceres**
Closes #192

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -46,12 +46,12 @@
     "@types/compression": "^1.7.0",
     "@types/cors": "^2.8.10",
     "@types/express": "^4.17.11",
-    "@types/jest": "^26.0.22",
+    "@types/jest": "^26.0.23",
     "@types/morgan": "^1.9.2",
-    "@types/node": "^14.14.37",
-    "@types/node-fetch": "^2.5.8",
-    "@types/split2": "^2.1.6",
+    "@types/node": "^15.0.2",
```
Comment (sidvishnoi) [inline package.json:51]: But we're using node 14 🤔

Comment (marcoscaceres) [reply package.json:51]: More reason to move to 16.

Comment (sidvishnoi) [reply package.json]: Sent https://github.com/w3c/respec-web-services/pull/197

**Review by sidvishnoi — CHANGES_REQUESTED** [review]
Wait for https://github.com/w3c/respec-web-services/pull/197

**Comment by sidvishnoi** [comment]
Can you run ncu again? With npm7 so it uses new lockfile format.

**Comment by marcoscaceres** [comment]
Not a big deal, but I remembered why we were using 14.x.... basically, ReSpec itself targets 14 because LTS, so it was more sane (locally) to use a single version. 

Running now...

**Comment by marcoscaceres** [comment]
We might need to add `.nvm` to Respec itself too... but it just uses lts

**Comment by marcoscaceres** [comment]
ah crap... I made a boo boo...

**Pull request closed without merging**